app/src/main/java/ru/yandex/praktikumchatapp/MainActivity.kt

```diff
@@ -46,32 +47,26 @@ class MainActivity : ComponentActivity() {
         enableEdgeToEdge()
         setContent {
             PraktikumChatAppTheme {
-                Scaffold(
-                    modifier = Modifier.fillMaxSize(),
-                    topBar = {
-                        TopAppBar(
-                            title = {
-                                Text(stringResource(R.string.app_name))
-                            }
-                        )
-                    },
-                    content = { innerPadding ->
-                        ChatScreen(
-                            modifier = Modifier.padding(innerPadding)
-                        )
-                    }
-                )
+                Scaffold(modifier = Modifier.fillMaxSize(), topBar = {
+                    TopAppBar(title = {
+                        Text(stringResource(R.string.app_name))
+                    })
+                }, content = { innerPadding ->
+                    ChatScreen(
+                        modifier = Modifier.padding(innerPadding)
+                    )
+                })
             }
         }
     }
 }
 
 @Composable
 fun ChatScreen(
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
 ) {
     val viewModel = remember { ChatViewModel() }
-    val messagesList = viewModel.messages.observeAsState(emptyList())
+    val messagesList = viewModel.messages.collectAsState()
```

Comment:
У `collectAsState` есть проперти делегат, который позволяет не писать везде `.value`: 
```suggestion
    val messagesList by viewModel.messages.collectAsState()
```